CHANGELOG.md

```diff
@@ -5,15 +5,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### Changed
 - update dependencies in package,json
 - changed color values of various css elements to enhance accessibility
 - recolor of navbar-inverse
+- include svg-logo in index
```

Comment:
Missing reference, wording:

Missing reference:
It is hard to backtrack changes to their requirements when no links or references are given. If I get it right you base this PR on issue #16 (whose description could be more tacky as well, but one step after another).

Wording:
I miss in this changelog what actually depicts the logo being replaced, namely a "Cloudogu" logo as there may be a myriad of logos.

How about this suggestion?
```suggestion
- include Cloudogu logo in index as SVG #16 
```